Code/Editor/EditorModularViewportCameraComposer.cpp

```diff
@@ -172,19 +174,29 @@ namespace SandboxEditor
             return SandboxEditor::CameraScrollSpeed();
         };
 
+        auto pivotFn = []
+        {
+            // use the manipulator transform as the pivot point
+            AZStd::optional<AZ::Transform> entityPivot;
+            AzToolsFramework::EditorTransformComponentSelectionRequestBus::EventResult(
+                entityPivot, AzToolsFramework::GetEntityContextId(),
+                &AzToolsFramework::EditorTransformComponentSelectionRequestBus::Events::GetManipulatorTransform);
+
+            // otherwise just use the identity
+            return entityPivot.value_or(AZ::Transform::CreateIdentity()).GetTranslation();
```

Comment:
if you check use ```has_value``` instead of ```value_or``` you save the creation of the temp Identity transform needed to return the translation vector.
```
if(entityPivot.has_value())
{
    return entityPivot.value().GetTranslation();
}
return AZ::Vector3::CreateZero();
```